cmd/minikube/cmd/start.go

```diff
@@ -1267,25 +1267,30 @@ func validateDiskSize(diskSize string) error {
 }
 
 // validateRuntime validates the supplied runtime
-func validateRuntime(runtime string) error {
+func validateRuntime(rtime string) error {
 	validOptions := cruntime.ValidRuntimes()
 	// `crio` is accepted as an alternative spelling to `cri-o`
 	validOptions = append(validOptions, constants.CRIO)
 
 	var validRuntime bool
 	for _, option := range validOptions {
-		if runtime == option {
+		if rtime == option {
 			validRuntime = true
 		}
 
 		// Convert `cri-o` to `crio` as the K8s config uses the `crio` spelling
-		if runtime == "cri-o" {
+		if rtime == "cri-o" {
 			viper.Set(containerRuntime, constants.CRIO)
 		}
+
+	}
+
+	if (rtime == "crio" || rtime == "cri-o") && (strings.HasPrefix(runtime.GOARCH, "ppc64") || detect.RuntimeArch() == "arm" || strings.HasPrefix(detect.RuntimeArch(), "arm/")) {
+		out.Error(reason.RuntimeIncompat, "The {{.runtime}} runtime is not compatible with the {{.arch}} architecture.", out.V{"runtime": rtime, "arch": runtime.GOARCH})
```

Comment:
how about  we just exit here and link to github issue on CRIO github ?

Reply:
returning an error will effectively exit here, changed this to returning an error and added a link to the issue